src/qmqtt_client_p.cpp

```diff
@@ -93,19 +93,7 @@ void QMQTT::ClientPrivate::init(const QString& hostName, const quint16 port, con
 
     _hostName = hostName;
     _port = port;
-    if (ssl)
```

Comment:
The ssl commit should not be part of the pull request. Can you ignore it?
The other commit was included by accident.
I can create another pull request is you want.

Regards,
Ernst

On Aug 25, 2016 9:21 AM, "Matthias Dieter Wallnöfer" <
notifications@github.com> wrote:

> In src/qmqtt_client_p.cpp
> <https://github.com/emqtt/qmqtt/pull/76#discussion_r76192255>:
>
> > @@ -93,19 +93,7 @@ void QMQTT::ClientPrivate::init(const QString& hostName, const quint16 port, con
> >
> >      _hostName = hostName;
> >      _port = port;
> > -    if (ssl)
>
> I do not like this part of the patch. You just drop the if (ssl) part and
> call always Network(ssl, ignoreSelfSigned), also if it might not have
> been defined at all (#ifdef in src/qmqtt_network.h).
>
> I would stuck with my code and just change line 99 into _network.reset(new
> Network(ssl, ignoreSelfSigned));, so we do not break anything.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/emqtt/qmqtt/pull/76/files/5fde7981f33a6540c1d4404993b9e4b9d0b99dd6#r76192255>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARpGJUuuH_VyqRuXZtU6IIlG0o45LiQDks5qjUJ2gaJpZM4JsTqv>
> .
>